tests/OriginInvestorsClaim.test.js

```diff
@@ -334,7 +348,7 @@ contract("OriginInvestorsClaim", (accounts) => {
             );
         });
 
-        it("should create vesting contract within vesting period", async () => {
+        it("should be able to sert investor initial amount list", async () => {
```

Comment:
Apparently, we need to update the OriginInvestorsClaim contract (because initially it was using cliff & duration from staking contract), meanwhile when trying to get the vesting, it's using the cliff & duration from lockedSOV

updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/540/commits/4075c453d4bda8c4b6c4f16b9ca1d8f3dc4e75b9)